### PR TITLE
Add home screen

### PR DESCRIPTION
**Pull request by UtsavK55**
This PR consists of home screen.

### CODE REVIEW

File: src/screens/home/Home.tsx [hunk 1]
```diff
@@ -1,11 +1,106 @@
...
+
+      const recipeData = data?.recipes?.map(
+        ({id, title, image, vegetarian}: RecipeCard) => ({
+          id,
+          title,
+          image,
+          vegetarian,
+        }),
+      );
+      console.log(recipeData);
```
Comment (02priyanshi) [inline src/screens/home/Home.tsx]: remove consoles

File: src/screens/home/Home.tsx [hunk 2]
```diff
@@ -1,11 +1,106 @@
...
+  const getAllData = async () => {
+    randomRecipeTypes.map(async type => {
+      let url = randomRecipeUrl(type);
+      if (selectedCuisine) {
+        url += `&cuisine=${selectedCuisine}`;
+      }
+      if (selectedDiet) {
+        url += `&diet=${selectedDiet}`;
+      }
+      console.log(url);
```
Comment (02priyanshi) [inline src/screens/home/Home.tsx]: remove console

File: src/screens/home/Home.tsx (hunk 1 above)
Comment (adiTrivsme) [inline src/screens/home/Home.tsx]: remove logs

File: src/screens/home/Home.tsx [hunk 3]
```diff
@@ -1,11 +1,106 @@
...
+  const homeNavigation = useNavigation<HomeScreenNavigationType>();
+
+  const [appetizers, setAppetizers] = useState<AllRecipeCards>([]);
+  const [mainCourse, setMainCourse] = useState<AllRecipeCards>([]);
+  const [desserts, setDesserts] = useState<AllRecipeCards>([]);
+  const [selectedCuisine, setSelectedCuisine] = useState<string | null>(null);
+  const [selectedDiet, setSelectedDiet] = useState<string | null>(null);
+
+  const getAllData = async () => {
+    randomRecipeTypes.map(async type => {
```
Comment (02priyanshi) [inline src/screens/home/Home.tsx:28]: refactor this code

**Review by 02priyanshi — CHANGES_REQUESTED** [review]

**Review by 02priyanshi — APPROVED** [review]

**Pull request merged**